content/en/building/reference/api.md

```diff
@@ -874,10 +874,10 @@ Content-Type: application/json; charset=utf-8
 }
 ```
 
-2. Get 3 people of type `person` by using cursor `1`.
+2. Get the first 3 people of type `person` by using cursor `0`.
```

Comment:
So, _technically_ passing `cursor=0` will give you the "first" people, but that is just because of an implementation detail of how we have implemented paging (and one that might change in future releases).

The actual value of a cursor is only promised to be a string "token" value that _the server_ will interpret to know which page to return. Consumers should only pass in token values that were returned from the server on previous calls to the same API. If you want the "first" page of people, you should not pass any cursor.

Reply:
ah - gotcha - I'm down with being technically accurate!

However, you could still miss a record (first or otherwise) but not starting at zero I believe.  Let's go with:

```suggestion
2. Get 3 people of type `person` by using cursor `0`.
```

yeah?

Reply:
:thinking: Still not a fan of using `0` for the example cursor value.  I do now see your concern though with putting `cursor=1` in the example request since it may not be clear that things are 0-based and this is not where things should start.  What do you think about using `cursor=100`?  The previous example does not pass any cursor/limit params so it will return the first 100 persons.  Then, to get the next persons, you would pass `cursor=100`.

Reply:
oh - maybe?  I'm surprised about how hard this one change is :sweat_smile: 

Fixed in 5e5260d - thoughts?